doc/whats_new/v0.23.rst

```diff
@@ -319,6 +319,13 @@ Changelog
   random noise to the target. This might help with stability in some edge
   cases. :pr:`15179` by :user:`angelaambroz`.
 
+- |Efficiency| Speed up :class:`linear_model.MultiTaskLasso`,
```

Comment:
I'm not sure this can make it to 0.23

@adrinjalali this is ready for a merge, so your choice basically ;)